### PR TITLE
Syncdev

### PR DESCRIPTION
**Pull request by ShreyaGangishetty**
*Issue #, if available:*

*Description of changes:*

merging master into develop

*Description of how you validated changes:*
make pr is successful

*Checklist:*

- [x] `make pr` passes

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: examples/apps/api-gateway-multiple-origin-cors/package-lock.json [hunk 1]
```diff
@@ -3339,9 +3339,9 @@
...
     "mixin-deep": {
-      "version": "1.3.1",
-      "resolved": "https://registry.npmjs.org/mixin-deep/-/mixin-deep-1.3.1.tgz",
-      "integrity": "sha512-8ZItLHeEgaqEvd5lYBXfm4EZSFCX29Jb9K+lAHhDKzReKBQKj3R+7NOF6tjqYi9t4oI8VUfaWITJQm86wnXGNQ==",
+      "version": "1.3.2",
+      "resolved": "https://registry.npmjs.org/mixin-deep/-/mixin-deep-1.3.2.tgz",
+      "integrity": "sha512-WRoDn//mXBiJ1H40rqa3vH0toePwSsGb45iInWlTySa+Uu4k3tYUSxa2v1KqAiLtvlrSzaExqS1gtk96A9zvEA==",
       "dev": true,
       "requires": {
         "for-in": "^1.0.2",
```
Comment (keetonian) [inline examples/apps/api-gateway-multiple-origin-cors/package-lock.json]: Is this change necessary?

Comment (ShreyaGangishetty) [reply examples/apps/api-gateway-multiple-origin-cors/package-lock.json]: This change was directly pushed to master #1213. This was the request from bot.
